ios/FluentUI/People Picker/InitialsView.swift

```diff
@@ -131,7 +133,11 @@ class InitialsView: UIView {
 
     override func layoutSubviews() {
         super.layoutSubviews()
-        initialsLabel.frame = bounds
+        let width = bounds.width - (avatarSize.insideBorder * 2.5)
```

Comment:
where is this 2.5 and 1.25 coming from?

Reply:
Currently, the text is not adjusting because it is just barely within the frame for the extra small case (if you zoom in you can see that there is no overlap, but the text is just touching the interior of the bounds. To accommodate for that, the 0.5 provides slight buffer so that the text can adjust accordingly. 1.25 is half of 2.5 which is in consequence of the width and height adjustment

Reply:
if these paddings are coming from design then the number should be define as constants then do the 1.25/2 and such for readability for the next person? I also think we should write comments for more clarity for the future engineer reading this code

Reply:
and this is going to be true for all sizes? or does this bound calculation should only happen for extra small case?

Reply:
Ok! And yes this applies to all sizes if the case ever arises where the text is the same/exceeds the size of its containing bounds. However currently the only case that ends up getting affected by the text size adjustment is the extra small case (since the other font sizes are already well within the containing bounds)